3_visualize.R

```diff
@@ -0,0 +1,6 @@
+
+p3_targets_list <- list(
+  # Render data summary report
+  tarchetypes::tar_render(p3_wqp_spC_report, "3_visualize/src/report-wqp-salinity-data.Rmd",output_dir = "3_visualize/out")
```

Comment:
It might help to note in a comment that tar_render() returns a `format = 'file'` target